### ISSUE TITLE
Bug: eqMac crashes every time output is EVO4

### ISSUE THREAD

**Issue by hzenginx** [bitgapp/eqMac#558]
**Disclaimer:** 
- [x] I have checked for a similar issue and sure it hasn't been reported before.

**Describe the bug**
I have a Audient EVO4 audio interface, whenever I use it as the main output of my system, eqMac fails to start. When I open eqMac while using a different output; eqMac works as long as I don't choose my audio interface as the output(in eqMac); if I do it crashes.

**Steps to Reproduce**
1. Own a Audient EVO4
2. Set system audio to this device
2. Open eqMac 
4. It crashes

Alternative:
1. Set system audio to any device other than EVO4
2. Open eqMac
3. Choose EVO4 as the output in eqMac
4. It crashes

**Expected behaviour**
It shouldn't crash :)

**Setup information:**
 - Audio device used for playback: Audient EVO 4 Audio Interface
 - Audio transmission interface: Connected with usb-c
 - macOS Version: Big Sure latest version, also 12.0 Beta 4
 - eqMac Version 1.2.0
 - eqMac driver Version 1.2.3

**Additional information**
I created a agg. midi device with only EVO4 in it and eqMac works with it. Just a workaround.

Crash log when starting eqMac from terminal:

```
ERROR: SentryCrashCPU.c (58): _Bool sentrycrashcpu_i_fillState(const thread_t, const thread_state_t, const thread_state_flavor_t, const mach_msg_type_number_t): thread_get_state: (ipc/send) invalid destination port
ERROR: SentryCrashCPU.c (58): _Bool sentrycrashcpu_i_fillState(const thread_t, const thread_state_t, const thread_state_flavor_t, const mach_msg_type_number_t): thread_get_state: (ipc/send) invalid destination port
ERROR: SentryCrashMachineContext.c (232): void sentrycrashmc_resumeEnvironment(): thread_resume (0000ab27): (os/kern) failure
[1]    4258 trace trap  ./Contents/MacOS/eqMac
```

Crash log when I start it with a workaround and change the output to EVO4:

```
2021-08-30 20:07:09.6150 eqMac (Source/Application.swift:171) outputChanged:  EVO4 (89)  starting PlayThrough
ERROR: SentryCrashCPU.c (58): _Bool sentrycrashcpu_i_fillState(const thread_t, const thread_state_t, const thread_state_flavor_t, const mach_msg_type_number_t): thread_get_state: (ipc/send) invalid destination port
ERROR: SentryCrashCPU.c (58): _Bool sentrycrashcpu_i_fillState(const thread_t, const thread_state_t, const thread_state_flavor_t, const mach_msg_type_number_t): thread_get_state: (ipc/send) invalid destination port
ERROR: SentryCrashMachineContext.c (232): void sentrycrashmc_resumeEnvironment(): thread_resume (0000ae07): (os/kern) failure
```

**Comment by hzenginx** [comment]
Hey @nodeful,

Decided to build and fix it myself; on the master branch, bug is caused by [AudioDevice.swift:L68](https://github.com/bitgapp/eqMac/blob/master/native/app/Source/Extensions/AudioDevice.swift#L68).
EVO4 interface master channel is not mutable and channels from 1-there are 4- causes a null to be returned from `self.isMuted` which is force unwrapped causing a NPE. 

I tried with couple of other outputs I have and all of them returned a null.

Replacing it with a default value `false` solves the problem for the master branch; for latest release there is another bug when determining the sampling rate which is solved in master.

I could use my own build but since the reason I'm paying for the Pro version is AutoEQ Integration and I can't get it on my own build; how far you're away from a new release where we can introduce a fix for this?

**Comment by nodeful** [comment]
@hzenginx thanks for looking into this, I agree with your logic, unfortunately I never had a device that wouldn't have a Mutable channel. Do you have a fork of the repo where you've fixed the issues you're having, especially the issue with the sampling rate you've mentioned. I'm now fixing up last little bugs before release of v1.3.0 where this fix should be included

**Comment by hzenginx** [comment]
raised the CR. 

sampling rate bug was already solved in master.

**Comment by nodeful** [comment]
Yeap thanks! I actually done that fix with Mute just haven't committed 😄 Hopefully will release the update tomorrow

**Comment by hzenginx** [comment]
Nice, thanks!

**Comment by nodeful** [comment]
As the fix for this bug has been release in v1.3.0 I will close this issue. 
If this issue still persists please let me know.
Thank you!
